### ISSUE TITLE
Push a new build to Docker Hub and set up autobuilds

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap-docker-base#19]

**Comment by fingolfin** [comment]
Alternatively, perhaps automated builds can be reenabled? I read through issue #3 but I don't find the claims there conclusive. I also should point out that the build was using `make -j` and that by itself caused random errors for me, which is one of the reasons I removed it here. In general, my long-year experience is that using `make -j` is almost always WRONG -- and in fact can be slower than using `make -jN` for well-chosen `N`, because launching hundreds of jobs in parallel quickly overwhelm IO, memory and CPU resources of most regular machines, and definitely of any free VM available on Docker Hub, Travis, etc. etc.

**Comment by olexandr-konovalov** [comment]
It definitely won't hurt to try. The process might well be now faster and less memory consuming. E.g. polymake is installed as an ubuntu package and not from the source. Maybe there are other simplifications as well...

**Comment by olexandr-konovalov** [comment]
I've managed to build https://cloud.docker.com/u/gapsystem/repository/docker/gapsystem/gap-docker-base on DockerHub. It took 90 minutes. Apparently the time limits for these builds are now higher than when we tried that for the first time. @sebasguts should be happy that there is no more pressure to do it manually!

The remaining task is now to set up autobuilds on Docker Hub so that any change in this repository will trigger a new build.
